common/src/main/java/com/wynntils/models/horse/HorseModel.java

```diff
@@ -30,6 +31,52 @@ public Optional<HorseItem> getHorse() {
         return Models.Item.asWynnItem(McUtils.inventory().getItem(horseSlot), HorseItem.class);
     }
 
+    public Optional<Integer> calculateNextLevelSeconds() {
+        if (getHorse().isEmpty()) return Optional.empty();
+
+        HorseItem horseItem = getHorse().get();
+
+        if (horseItem.getLevel() == CappedValue.EMPTY || horseItem.getXp() == CappedValue.EMPTY)
+            return Optional.empty();
```

Comment:
Don't forget to use brackets here, we only don't use them if it all fits on one line for control statements

Reply:
Also for this, splotlessApply is moving the return onto a new line. Is there something I'm not understanding about it?

Reply:
Spotless will move it just because it extends past the max line length but it's not able to add the brackets